t/live.t

```diff
@@ -466,6 +466,14 @@ Customers: {
             );
             $subs_list = $stripe->list_subscriptions(customer => $customer);
             is scalar @{$subs_list->data}, 2, 'Customer now has two subscriptions';
+
+            eval {
+                $stripe->post_subscription(
+                    customer     => $customer->id,
+                    subscription => $subs_list
+                );
+            };
+            ok $@, "subscription should be of type Net::Stripe::Subscription";
```

Comment:
This is a good test, but only of the failure case. Would it be possible to add tests for the two cases with the :-1:  above?